libs/blocks/section-metadata/section-metadata.css

```diff
@@ -189,6 +189,20 @@ main > .section[class*='-up'] > .content {
   margin: 0;
 }
 
+.section.masonry-layout {
```

Comment:
I think along the same lines as the `basic-brick` conversation from Chris M, I think `masonry-layout` can be simplified to just `masonry`. Layout is implied unless there are more options related to `masonry`.

Reply:
@rgclayton Adding the update to the comment - masonry-layout is a polyfill and the authoring does not require the authors to add this as a style. As per the latest authoring format this would only need new masonry key row in the section-metadata with layout info.